Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -742,6 +742,15 @@ def audit_revision_and_version_scheme
     return if @new_formula
 
     fv = FormulaVersions.new(formula)
+
+    previous_version_and_checksum = fv.previous_version_and_checksum("origin/master")
+    [:stable, :devel].each do |spec_sym|
+      next unless spec = formula.send(spec_sym)
+      next unless previous_version_and_checksum[spec_sym][:version] == spec.version
+      next if previous_version_and_checksum[spec_sym][:checksum] == spec.checksum
+      problem "#{spec_sym}: only sha256 changed; needs to be confirmed by the developer"
```

Comment:
Oh, right, I was thinking this was HBC-only (we have the documentation for this case). Never mind, then.